src/app/components/payment-dialog/payment-dialog.component.ts

```diff
@@ -75,7 +89,70 @@ export class PaymentDialogComponent implements OnInit {
                     : undefined
         };
 
-        this.dialogRef.close(payload);
+        this.pendingTransferPollingService.pendingTransfers$
+            .pipe(
+                first(),
+                switchMap(pendingTransfers => {
+                    const pendingPayment = pendingTransfers.find(
+                        pendingTransfer =>
+                            pendingTransfer.token_address ===
+                                payload.tokenAddress &&
+                            pendingTransfer.role === 'initiator' &&
+                            pendingTransfer.locked_amount.isEqualTo(
+                                payload.amount
+                            ) &&
+                            pendingTransfer.target === payload.targetAddress &&
+                            (payload.paymentIdentifier
+                                ? pendingTransfer.payment_identifier.isEqualTo(
+                                      payload.paymentIdentifier
+                                  )
+                                : true)
+                    );
+
+                    if (pendingPayment) {
+                        const token = this.raidenService.getUserToken(
+                            payload.tokenAddress
+                        );
+                        const formattedAmount = amountToDecimal(
+                            payload.amount,
+                            payload.decimals
+                        ).toFixed();
+                        const confirmationPayload: ConfirmationDialogPayload = {
+                            title: 'Retrying payment',
+                            message:
+                                `There is already a payment of <b>${formattedAmount} ${
+                                    token.symbol
+                                }</b> being sent to <b>${
+                                    payload.targetAddress
+                                }</b>. <br/>` +
```

Comment:
`<strong>...</strong>` instead of `<b>...</b>`

Reply:
Makes sense. I used `<b>` because it is used in every other confirmation dialog. But I will change that.